CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # avro-builder changelog
 
+## v1.1.0 (unreleased)
+- Add support for Avro 1.11
+- Add support for decimal logical types.
```

Comment:
Can we document which logical types avro-builder supports in the README?

Reply:
I've updated the README to try and clarify this.

Since logical type support was added to avro-builder it has been very permissive about the specification of logical types. Logical type values are not validated, so values that are not in the spec an be used. I don't know if custom logical types have ever been used, but in theory users could apply their own interpretation, and new types can be supported without updating this library.

The decimal logical type is unique in that it is the only logical type that requires additional attributes on the type. For all other logical types, just the name of the logical type is enough.